python/shotgun_model/data_handler_cache.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2016 Shotgun Software Inc.
```

Comment:
Thanks for abstracting out this piece of code. It really makes the model simpler to read! :)